ReactiveCocoa/UIKit/iOS/BindingTargetSubstituting.generated.swift

```diff
@@ -0,0 +1,194 @@
+// This is auto-generated by gyb.
+
+import Foundation
+import ReactiveSwift
+import enum Result.NoError
+import UIKit
+
+extension UIActivityIndicatorView {
+
+	// isAnimating: Bool
+
+	public static func <~ <Source: BindingSourceProtocol>(
+		target: UIActivityIndicatorView,
+		source: Source
+	) -> Disposable? where Source.Value == Bool, Source.Error == NoError {
+		return target.reactive.isAnimating <~ source
+	}
+
+}
+extension NSLayoutConstraint {
+
+	// constant: CGFloat
+
+	public static func <~ <Source: BindingSourceProtocol>(
+		target: NSLayoutConstraint,
+		source: Source
+	) -> Disposable? where Source.Value == CGFloat, Source.Error == NoError {
+		return target.reactive.constant <~ source
+	}
```

Comment:
This one feels weird to me. I don't think it's obvious that it'd be targeting the content. With something that's essentially a value, defaults make sense. But I don't think this fits in that category.